Tasks/XamariniOSV2/package.json

```diff
@@ -17,8 +17,13 @@
   },
   "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
   "dependencies": {
-    "@types/node": "^6.0.101",
-    "azure-pipelines-task-lib": "^2.9.3",
+    "@types/mocha": "^5.2.7",
+    "@types/node": "^10.17.0",
+    "@types/uuid": "^8.3.0",
+    "azure-pipelines-task-lib": "^3.0.6-preview.0",
     "msbuildhelpers": "file:../../_build/Tasks/Common/msbuildhelpers-1.0.0.tgz"
```

Comment:
Looks like this packages has "azure-pipelines-task-lib": "^2.9.3" as a dependency. Can this cause any issues?

Reply:
Thanks! 
I'm not sure, need your advice here
+ @anatolybolshakov , @egor-bryzgalov 
Tests are passing locally and in the PR builds. Task also works as expected on a test environment

BTW, it looks like we have the same situation here: https://github.com/microsoft/azure-pipelines-tasks/pull/13898/files

Reply:
Regarding task-lib version - agree, we need to update msbuildhelpers package also - as we did previously for other packages.
Let's try approach which we used in this PR - to resolve issue with tests timeout.

Reply:
Npm package _azure-pipelines-tasks-msbuildhelpers_ package was published. Used it instead of common _msbuildhelpers_ package

BTW, there were no issues with incompatibility of task-lib versions, since task-lib was deleted from msbuildhelpers node-modules due tp make.json configuration:
```    "rm": [
         {
            "items": [
                "node_modules/msbuildhelpers/node_modules/azure-pipelines-task-lib"
            ],
            "options": "-Rf"
        }
    ]
```